src/main/java/expression/tokenextractor/operand/ListOperandExtractor.java

```diff
@@ -0,0 +1,39 @@
+package expression.tokenextractor.operand;
+
+import expression.myexception.InvalidTokenException;
+import expression.tokenextractor.TokenExtractor;
+import expression.token.operand.ListOperand;
+import expression.token.Token;
+import expression.tokenizer.TokenizerImpl;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ListOperandExtractor implements TokenExtractor {
+
+  final String regex = "^\\( *((?:(?:[\"0-9A-Za-z]+)* *, *)* *(?:[\"0-9A-Za-z]+)) *\\)";
```

Comment:
This can be a really hard to read and maintain regex. Could use some grammar to parse complex tokens